src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -95,9 +95,18 @@
 import json
 import logging
 import os
+
+import yaml
+
```

Comment:
as yaml is in stdlib could we not add a line break before importing it?

Reply:
alright this is the result from lint.
Made without \n and still got the above